frontend/src/components/UserUpdateSfdsAuth.vue

```diff
@@ -145,8 +150,12 @@ export default {
     };
   },
   async created() {
+    //TODO error handling
```

Comment:
I guess this TODO is outside of the scope of this change? It'll be a rare but outages to the SFDS application would bring down the API, so how would the UMC behave if this were the case?

Reply:
Errors are actually handled, caught and logged by the getUses and getMailbox methods